royalty/queen

```diff
@@ -0,0 +1,4 @@
+Queen - ()
+- Takes the place of the King, if they are alive and there is no alive Prince when the King dies.
+- Upon death, if the cause of death is anything other than tribute, the Chief Warrior of their team will gain and extra attack the following night.
```

Comment:
type here, "will gain and extra attack"

Reply:
fix the typo and make the first line phrased better. something like
"If the King dies, a living Queen will become the new King. Prince takes priority over Queen if both are alive."

Reply:
also needs a power rating before i can merge

Reply:
One question, what do you think the power level of Queen should be? It seems relatively powerful, however the ability only activates upon death to anything other than tribute, which of course, has the risk of the King dying in that time, causing the Queen to also die if no Prince is present.

Reply:
Also, the ability is basically null if the Chief Warrior is respawning during that time.

Reply:
@12Metatron Please do let us know what power level the Queen should have

Reply:
Looks good to me